[2021-05-12 09:03:55,159] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: email_test.print_date 2021-05-11T00:00:00+00:00 [queued]>
[2021-05-12 09:03:55,165] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: email_test.print_date 2021-05-11T00:00:00+00:00 [queued]>
[2021-05-12 09:03:55,165] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 09:03:55,165] {taskinstance.py:1069} INFO - Starting attempt 1 of 11
[2021-05-12 09:03:55,165] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-12 09:03:55,295] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): print_date> on 2021-05-11T00:00:00+00:00
[2021-05-12 09:03:55,297] {standard_task_runner.py:52} INFO - Started process 2120291 to run task
[2021-05-12 09:03:55,303] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'email_test', 'print_date', '2021-05-11T00:00:00+00:00', '--job-id', '17', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/email_test.py', '--cfg-path', '/tmp/tmpioh04w3_', '--error-file', '/tmp/tmpaokcszx5']
[2021-05-12 09:03:55,304] {standard_task_runner.py:77} INFO - Job 17: Subtask print_date
[2021-05-12 09:03:55,416] {logging_mixin.py:104} INFO - Running <TaskInstance: email_test.print_date 2021-05-11T00:00:00+00:00 [running]> on host sdn-k8s-b2-3-3.maas
[2021-05-12 09:03:55,550] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=davidwu5858@gmail.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=email_test
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2021-05-11T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-11T00:00:00+00:00
[2021-05-12 09:03:55,550] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-12 09:03:55,551] {bash.py:158} INFO - Running command: date
[2021-05-12 09:03:55,558] {bash.py:169} INFO - Output:
[2021-05-12 09:03:55,560] {bash.py:173} INFO - Wed May 12 09:03:55 UTC 2021
[2021-05-12 09:03:55,560] {bash.py:177} INFO - Command exited with return code 0
[2021-05-12 09:03:55,682] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=email_test, task_id=print_date, execution_date=20210511T000000, start_date=20210512T090355, end_date=20210512T090355
[2021-05-12 09:03:56,053] {taskinstance.py:1246} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-05-12 09:03:56,075] {local_task_job.py:146} INFO - Task exited with return code 0
